R/bundle.R

```diff
@@ -316,6 +316,21 @@ writeBundle <- function(bundleDir, bundlePath, verbose = FALSE) {
   utils::tar(bundlePath, files = NULL, compression = "gzip", tar = tarImplementation)
 }
 
+
+getTarImplementation <- function() {
+  # Check the RSCONNECT_TAR environment variable first. If that is unset, check
```

Comment:
The comment is inconsistent with the implementation.

Reply:
Fixed